docs/attributes-registry/system.md

```diff
@@ -1,178 +1,169 @@
-# System
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
 
-<!-- toc -->
+# System
 
-- [CPU attributes](#cpu-attributes)
-- [Memory attributes](#memory-attributes)
-- [Paging attributes](#paging-attributes)
-- [Filesystem attributes](#filesystem-attributes)
-- [Network attributes](#network-attributes)
-- [Process attributes](#process-attributes)
-- [Deprecated System Attributes](#deprecated-system-attributes)
+- [System](#system-attributes)
+- [System Cpu](#system-cpu-attributes)
+- [System Deprecated](#system-deprecated-attributes)
```

Comment:
nit: is it possible to make deprecated section to be the last one?

Reply:
Probably.  It wasn't easy without a custom sorting function so I left it for later.  Could we do that in a follow-up PR?  It *should* be a JINJA template-only change, but we can add a filter to weaver to make it simpler.

Reply:
sure!